virttest/storage.py

```diff
@@ -726,8 +761,11 @@ def save_image(self, params, filename, root_dir=None):
             backup_size = os.path.getsize(src)
         else:
             # TODO: get the size of block/remote images
-            backup_size += int(float(utils_numeric.normalize_data_size(
-                                     self.size, order_magnitude="B")))
+            if self.size:
+                backup_size += int(
+                    float(utils_numeric.normalize_data_size(
+                        self.size, order_magnitude="B"))
+                )
```

Comment:
These two changes seem to help for another issue, so please create a separate commit too.

Reply:
Tiny code change, can we merge it along with the current patch

Reply:
I mean by creating a separate commit, not a separate pull request.

Reply:
Sorry for that, I didn't notice that, change done now.